ProcessLib/LiquidFlow/CreateLiquidFlowMaterialProperties.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   CreateLiquidFlowMaterialProperties.cpp
+ *
+ *   Created on December 14, 2016, 1:20 PM
+ */
+
+#include "CreateLiquidFlowMaterialProperties.h"
+
+#include "BaseLib/ConfigTree.h"
+#include "BaseLib/reorderVector.h"
+
+#include "MeshLib/PropertyVector.h"
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+#include "MaterialLib/PorousMedium/Porosity/Porosity.h"
+#include "MaterialLib/PorousMedium/Storage/Storage.h"
+#include "MaterialLib/Fluid/FluidProperties/CreateFluidProperties.h"
+
+#include "MaterialLib/Fluid/FluidPropertyHeaders.h"
+#include "MaterialLib/PorousMedium/PorousPropertyHeaders.h"
+
+#include "LiquidFlowMaterialProperties.h"
+
+namespace ProcessLib
+{
+namespace LiquidFlow
+{
+class LiquidFlowMaterialProperties;
+
+std::unique_ptr<LiquidFlowMaterialProperties>
+createLiquidFlowMaterialProperties(
```

Comment:
The parameters in this function are documented (see Documentation/prj/processes/process/LIQUID_FLOW). The parameters of material models are given in the creating functions of the models.

Actually, the body of this function is exactly moved from the constructor of class LiquidFlowMaterialProperties.